### PR TITLE
fix: ruby 2.7 deprecations

### PR DESCRIPTION
**Pull request by dgodd**

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1,74 +1,110 @@
...
+Lint/DeprecatedOpenSSLConstant:
+  Enabled: true
+
+Lint/MixedRegexpCaptureTypes:
+  Enabled: true
+
+Lint/RaiseException:
+  Enabled: true
+
+Lint/StructNewOverride:
```
Comment (robotdana) [inline .rubocop.yml:88]: you can avoid this string of Enabled: true's by adding `NewCops: enable` to the AllCops block

File: lib/lightspeed/client.rb [hunk 2]
```diff
@@ -21,19 +23,23 @@ def accounts
...
     def get(**args)
-      perform_request(args.merge(method: :get))
+      args.merge!(method: :get)
+      perform_request(**args)
     end
 
     def post(**args)
-      perform_request(args.merge(method: :post))
+      args.merge!(method: :post)
+      perform_request(**args)
```
Comment (robotdana) [inline lib/lightspeed/client.rb:32]: this could be `perform_request(**args, method: :post)` etc

Comment (dgodd) [reply lib/lightspeed/client.rb]: Much nicer, Thanks. (and changed)

File: .rubocop.yml (hunk 1 above)
Comment (dgodd) [reply .rubocop.yml]: I kind of liked the explicitness of before, but I like the low maintenance of this. I've changed

**Pull request merged**